network/vpc/how-to/configure-a-public-gateway.mdx

```diff
@@ -124,6 +124,18 @@ You can review and (if you wish) modify the NAT configuration of an existing Pub
 
 Your new static NAT configuration is now saved, and displays on the NAT panel. You can repeat steps 6-8 to add new static NAT configurations as you wish.
 
+## How to enable SMTP
+
+By default, the SMTP ports (25, 465 and 587) on your Public Gateway are blocked to avoid spam. If you wish to send emails from resources located behind your Public Gateway, you should open these ports by enabling SMTP as follows:
```

Comment:
should we say that the resource behind the Public Gateway needs also to enable SMTP?

Reply:
How about:

By default, the SMTP ports (25, 465 and 587) on your Public Gateway are blocked to avoid spam. If you wish to send emails from resources located behind your Public Gateway, make sure that the resources behind the Public Gateway [have also enabled SMTP](/console/my-account/how-to/send-emails-from-your-instance/) and then enable SMTP on your Public Gateway as follows: